server/src/main/java/com/objectcomputing/checkins/services/permissions/Permission.java

```diff
@@ -47,7 +47,7 @@ public enum Permission {
   CAN_DELETE_REVIEW_PERIOD("Delete review period", "Review Period"),
   CAN_ADMINISTER_SETTINGS("Add or edit settings", "Settings"),
   CAN_VIEW_SETTINGS("View settings", "Settings"),
-  CAN_VIEW_ALL_PULSE_RESPONSES("View all pulse responses", "Pulse");
```

Comment:
I moved this permission to the Reporting category since it is a kind of report and we don't have anything else in the Pulse category.